### ISSUE TITLE
Add a smoke test for checking suitability of build for use as bootstrap

### ISSUE THREAD

**Issue by smlambert** [adoptium/temurin-build#3366]
Add a smoke test target into https://github.com/adoptium/temurin-build/tree/master/test/functional/ for checking whether a build is suitable as a bootstrap build for a later version.

**Comment by sxa** [comment]
Have you got the details of how you would determine that?

**Comment by smlambert** [comment]
No, would like to look at the test that is within Red Hat that does something like this, but do not have the details atm, just was told it exists.
